From: Cordell Bloor <cgmb@slerp.xyz>
Date: Tue, 25 Oct 2022 22:11:07 -0600
Subject: disable upstream package logic

Forwarded: not-needed
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -380,7 +380,7 @@
 set(CPACK_RPM_EXCLUDE_FROM_AUTO_FILELIST_ADDITION "/opt" "${ROCM_PATH}")
 
 find_file (DEBIAN debian_version debconf.conf PATHS /etc)
-if(DEBIAN)
+if(0)
   # Write copyright file
   file(WRITE "${CMAKE_BINARY_DIR}/copyright"
   "Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
